yarn-project/noir-contracts/src/libs/noir-aztec/src/state_vars/immutable_singleton.nr

```diff
@@ -23,7 +24,8 @@ impl<Note, N> ImmutableSingleton<Note, N> {
     fn initialise(self, mut context: Context, note: Note) -> Context {
         // Nullify the storage slot.
         let nullifier = self.compute_initialisation_nullifier();
-        context = context.push_new_nullifier(nullifier);
+        // TODO(suyash): Check with David
```

Comment:
Check if the initialisation nullifier's commitment is "empty".

Reply:
This looks right to me! It seems like a nullifier that is just meant to signify that something has been initialized without actually nullifying a commitment.